.github/workflows/ci-unit-tests.yaml

```diff
@@ -0,0 +1,36 @@
+name: Continuous Integration testing for Pull Requests
+
+on: [pull_request]
+
+jobs:
+  ci-unit-tests:
+    runs-on: ${{ matrix.os }}
+    name: Tests on ${{ matrix.os }} with ${{ matrix.node }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+        node:
+          - 8
+          - 10
+          - 12
+        # x86 builds are only meaningful for Windows
+        exclude:
+          - os: ubuntu-latest
+          architecture: x86
+          - os: macos-latest
+          architecture: x86
```

Comment:
You're not using architecture, so you don't need this exclusion. This was so that my Python workflows would load for x86 and x64 Python builds in Windows.